site/content/docs/5.3/helpers/colored-links.md

```diff
@@ -5,13 +5,30 @@ description: Colored links with hover states
 group: helpers
 aliases:
   - "/docs/helpers/colored-links/"
-toc: false
+toc: true
 ---
 
-You can use the `.link-*` classes to colorize links. Unlike the [`.text-*` classes]({{< docsref "/utilities/colors" >}}), these classes have a `:hover` and `:focus` state.
+## Link colors
 
+You can use the `.link-*` classes to colorize links. Unlike the [`.text-*` classes]({{< docsref "/utilities/colors" >}}), these classes have a `:hover` and `:focus` state. Some of the link styles use a relatively light foreground color, and should only be used on a dark background in order to have sufficient contrast.
+
+<!--Boosted mod: feature limited to primary and light, we don't loop over "theme-colors"-->
 {{< example >}}
-<a href="#" class="link-primary">Primary link</a>
-<a href="#" class="link-light">Light link</a>
+<p><a href="#" class="link-primary">Primary link</a></p>
+<p><a href="#" class="link-light">Light link</a></p>
 {{< /example >}}
 
+{{< callout info >}}
+{{< partial "callouts/warning-color-assistive-technologies.md" >}}
```

Comment:
Do we need it since we force bg on our colored links ?

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1854/commits/f25cd1a3811e887799cc46dd18b65908dfa26561